### PR TITLE
Cloudwatch result visualization

### PR DESCRIPTION
**Pull request by mostafa**

### CODE REVIEW

File: src/data/markdown/docs/01 guides/03 Results visualization/09 Amazon CloudWatch.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    http_req_sending...........: avg=25.77µs  min=5.84µs   med=21.77µs  max=8.58ms   p(90)=31.41µs  p(95)=35.12µs
+    http_req_tls_handshaking...: avg=0s       min=0s       med=0s       max=0s       p(90)=0s       p(95)=0s
+    http_req_waiting...........: avg=110.49ms min=108.61ms med=110.22ms max=128.21ms p(90)=112.15ms p(95)=113.76ms
+    http_reqs..................: 29990  99.922514/s
+    iteration_duration.........: avg=110.79ms min=108.78ms med=110.44ms max=225.55ms p(90)=112.39ms p(95)=114.03ms
+    iterations.................: 29990  99.922514/s
+    vus........................: 15     min=15 max=15
+    vus_max....................: 15     min=15 max=15
+```
+
```
Comment (ppcano) [inline src/data/markdown/docs/01 guides/03 Results visualization/09 Amazon CloudWatch.md:153]: IMO, this is unnecessary.

File: src/data/markdown/docs/01 guides/03 Results visualization/09 Amazon CloudWatch.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+- Run the CloudWatch agent
+- Run the k6 test
+- Visualize k6 metrics in Amazon CloudWatch
+
+## Prerequisites
+
+- [k6](https://k6.io/open-source), an open-source load testing tool for testing the performance of APIs, microservices, and websites.
+- [Amazon EC2](https://aws.amazon.com/ec2/), an scalable virtual server solution to quickly have an up and running environment for your applications.
+- [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/), a monitoring and observability solution.
```
Comment (ppcano) [inline src/data/markdown/docs/01 guides/03 Results visualization/09 Amazon CloudWatch.md:18]: I wouldn't add prerequisites to follow the same format than the other guides.

File: src/data/markdown/docs/01 guides/03 Results visualization/09 Amazon CloudWatch.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+## Install k6
+
+For more information about how to install k6, please visit our [installation](https://k6.io/docs/getting-started/installation) guide. For Amazon EC2 instances running Debian 10 (Buster), you also need to install [GnuPG](https://www.gnupg.org/) software to verify repository signature before attempting to install k6 from the official repository:
+
+```bash
+sudo apt install -y gnupg
+```
+
```
Comment (ppcano) [inline src/data/markdown/docs/01 guides/03 Results visualization/09 Amazon CloudWatch.md:27]: On the other guides, we don't give instructions to install k6.

It's assumed that you are running k6 in your local machine. Which it makes my next question, why do I need to install k6 in EC2?

I think the installation should go into the details with EC2.

1 - Start with install the CloudWatch agent. (See the instructions)
2 - configure the CloudWatch agent like ...
3 - Start the agent (
4 - Run the test and output the result to the agent.


This is the structure of the instructions for ->  
-> https://k6.io/docs/results-visualization/new-relic
- > https://k6.io/docs/results-visualization/datadog

**Review by ppcano — CHANGES_REQUESTED** [review]

File: src/data/markdown/docs/01 guides/01 Getting started/04 Results Output.md [hunk 4]
```diff
@@ -106,7 +106,8 @@ The list of output plugins are:
 | [InfluxDB](/results-visualization/influxdb-+-grafana) | `k6 run --out influxdb`                                                         |
 | [JSON](/results-visualization/json)                   | `k6 run --out json`                                                             |
 | [StatsD](/results-visualization/statsd)               | `k6 run --out statsd`                                                           |
-| [New Relic](/results-visualization/new-relic)         | `k6 run --out statsd`, see [page](/results-visualization/new-relic) for details |
+| [New Relic](/results-visualization/new-relic)         | `k6 run --out statsd` |
```
[the remark this answers is not in the document]
Comment (mostafa) [reply src/data/markdown/docs/01 guides/01 Getting started/04 Results Output.md:109]: Addressed.

**Pull request merged**